src/vmm/src/devices/virtio/block/virtio/io/sync_io.rs

```diff
@@ -22,25 +22,36 @@ pub enum SyncIoError {
 
 #[derive(Debug)]
 pub struct SyncFileEngine {
-    file: File,
+    // 65536 is the largest buffer a linux guest will give us, empirically
```

Comment:
what did that empiric approach constitute?

Reply:
Can update the comment, sure.

> also, what will happen if it turns out insufficient? Will we detect that?

`MaybeBounce` will break larger request into chunks of size `N`, so there's no functional issue if that happens